plugins/repoclosure.py

```diff
@@ -103,6 +103,9 @@ def _get_unresolved(self, arch=None):
 
         if self.opts.pkglist:
             to_check = to_check.filter(name=self.opts.pkglist)
```

Comment:
Believe that each of argument should be checked separately. Otherwise the command like "dnf repoclosure --pkg dnf --pkg notMe" will exit with 0, even if package  "notMe" is not present.